release_notes/371-release-notes.adoc

```diff
@@ -0,0 +1,225 @@
+
+* *Images at most risk*: Provides a list of images and their risk priority, number of critical CVEs, and number of important CVEs. Click an image name to view more information. Use the *Options* menu to display fixable CVEs or all CVEs and display active images or all images on the dashboard.
+
+* *Deployments at most risk*: Provides the information that was previously available in the *Top risky deployments* widget, but displays additional information such as the resource location (cluster and namespace) and the risk priority score. Additionally, you can click on a deployment to view risk information about the deployment; for example, its policy violations and vulnerabilities. 
+
+* *Policy violations by category*: Provides a graphical representation of violations by predefined policy categories such as vulnerability management, security best practices, and privileges. This widget displays a color-coded bar graph of violations by severity level. You can hover over the bar for each category to view the number of violations. Click on a severity level at the bottom of the graph to add or remove that level from the display.
+
+* *Aging images*: This widget replaces the *Active violations by time* widget and allowing you to understand how many images have not been updated for a selected period of time. Images that are not updated can present more security risks than images that have been updated recently. You can configure your own age intervals or use the default ranges that {product-title-short} provides.  
+
+* *Compliance by standard*: The *Compliance* widget adds information to the Compliance widget that was provided in previous versions. It shows the status of various compliance benchmarks and the percentage of compliance coverage for each standard. Select *Options* to configure sorting by ascending or descending compliance percentage.
+
+The configurable widgets that displayed circle charts based on policy categories have been removed. To view similar data, you can use the *Policy violations by category* widget and select the number of violations in each category to display more information. 
+
+
+[id="network-policies-list"]
+=== List of network policies in Deployment tab for violations 
+
+When viewing a violation, you can select the *Deployment* tab and scroll to the *Network Policy* section to view a list of network policies in the namespace for the deployed cluster. For example, if you have configured the system to alert you of a violation of policy *Deployments should have at least one ingress network policy*, you can choose the *Deployment* tab to view a list of the existing network policies. You can use this list to verify which network policies exist in the namespace and determine whether or not an ingress network policy is missing. 
```

Comment:
Regarding whether or not: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#whether-or-not

I think it can just be "whether"